libraries/adaptive-expressions/src/builtinFunctions/accessor.ts

```diff
@@ -23,6 +23,9 @@ export class Accessor extends ExpressionEvaluator {
         super(ExpressionType.Accessor, Accessor.evaluator, ReturnType.Object, Accessor.validator);
     }
 
+    /**
```

Comment:
I don't think leaving this empty comments will be a good idea. Can we remove them? The easy way will be to remove the rule and fix it. We will help you on this (we'll do some pairing) :)